game-app/game-core/src/main/java/games/strategy/engine/framework/startup/mc/ServerModel.java

```diff
@@ -191,9 +190,7 @@ private void updatePlayersOnGameDataChanged(final GameData data) {
           playersToNodeListing.put(name, null);
         }
       } else {
-        Optional.ofNullable(messengers)
-            .ifPresent(
-                messenger -> playersToNodeListing.put(name, messenger.getLocalNode().getName()));
```

Comment:
Small consistency change. In practice messengers should always be non-null at this point,  but in case something odd happens it would now crash regardless of if the client is headless or not. Previously it would only cause an NPE on headless clients.